src/azure-cli-core/azure/cli/core/style.py

```diff
@@ -164,6 +164,40 @@ def format_styled_text(styled_text, theme=None):
     return ''.join(formatted_parts)
 
 
+def highlight_command(raw_command):
+    """Highlight a command to make it colored.
+
+    The highlighting rules after the command is splitted into args:
+        1. Args starting with '-' are regarded as parameter names, and they are with the ACTION style.
+        2. Args before the first parameter name are regarded as command name (or positional argument),
+        and they are with the ACTION style.
+        3. Args between the parameter names are regarded as parameter values, and they are with the PRIMARY style.
+        4. Args containing '=' are regarded as parameter name-value pairs, and they are with the PRIMARY style.
```

Comment:
```suggestion
    """Highlight a command with colors.

    For example, for 
        
        az group create --name myrg --location westus

    The command name 'az group create', argument name '--name', '--location' are marked as ACTION style.
    The argument value 'myrg' and 'westus' are marked as PRIMARY style.
    If the argument is provided as '--location westus', it will be marked as PRIMARY style.
```